apps/web/src/lang/fr.json

```diff
@@ -4403,6 +4419,10 @@
     "defaultMessage": "Les compétences comportementales désignent les principales qualités interpersonnelles et personnelles nécessaires pour occuper des emplois précis au sein de l’organisation. Ces compétences se rapportent généralement à la façon dont une personne agit, communique et interagit avec les autres.",
     "description": "Definition of behavioural skills"
   },
+  "R5sGKY": {
+    "defaultMessage": "Traiter l’information",
```

Comment:
I believe this is incorrect. The original string, Process information, is using the word process as a noun (**a process**) where as _traiter_ is the verb (**to process**).